testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -691,6 +691,9 @@ def pillar_get(key, minion)
   end
   node.run('rm -Rf /var/cache/salt/minion /var/run/salt /var/log/salt /etc/salt', false)
   step %(I disable the repositories "tools_update_repo tools_pool_repo" on this "#{host}" without error control)
+  if host.include? 'proxy'
+    step %(I disable the repositories "proxy_module_pool_repo proxy_product_pool_repo proxy_product_update_repo" on this "#{host}" without error control)
+  end
```

Comment:
LGTM. Just a sheldonish remark:

Now this steps mixes immediate commands from `node.run()` and calling other steps from `step %()`.

I would prefere to have only one type. And if possible, only immediate commands...

Reply:
Well, we only introduced a new step, but this was already mixing.
In any case, all those steps ends in a node.run call.